var/spack/repos/builtin/packages/ibmisc/package.py

```diff
@@ -65,14 +74,23 @@ class Ibmisc(CMakePackage):
 
     # Build dependencies
     depends_on('doxygen', type='build')
+    depends_on('doxygen', when='+doc', type='build')
 
     def cmake_args(self):
         spec = self.spec
-        return [
+        args = [
             '-DUSE_EVERYTRACE=%s' % ('YES' if '+everytrace' in spec else 'NO'),
+            '-DBUILD_PYTHON=%s' % ('YES' if '+python' in spec else 'NO'),
             '-DUSE_PROJ4=%s' % ('YES' if '+proj' in spec else 'NO'),
             '-DUSE_BLITZ=%s' % ('YES' if '+blitz' in spec else 'NO'),
             '-DUSE_NETCDF=%s' % ('YES' if '+netcdf' in spec else 'NO'),
             '-DUSE_BOOST=%s' % ('YES' if '+boost' in spec else 'NO'),
             '-DUSE_UDUNITS2=%s' % ('YES' if '+udunits2' in spec else 'NO'),
-            '-DUSE_GTEST=%s' % ('YES' if '+googletest' in spec else 'NO')]
+            '-DUSE_GTEST=%s' % ('YES' if '+googletest' in spec else 'NO'),
+            '-DBUILD_DOCS=%s' % ('YES' if '+doc' in spec else 'NO')]
+
+        if '+python' in spec:
+            args.append(
+                '-DCYTHON_EXECUTABLE=%s' % spec['py-cython'].command.path)
```

Comment:
Does this actually work? I would've thought it would search for an executable called `py-cython`. Might need to override this in the `py-cython` package.

Reply:
I believe it should work, once this is merged:
https://github.com/spack/spack/pull/7863/files#diff-08f3350689f26cd5f2732ba210d45ecf

I was following the pattern found in `python/package.py`.  Admittedly less necesary here than with Python, where the exe could be called `python2` or `python3`.  But it still seemed like a pattern worth following.

Your thoughts?

Reply:
That's fine, we'll just have to wait for the other PR to be merged first.